_translator-files/po/zh_CN/Running-a-Server.po

```diff
@@ -193,111 +200,115 @@ msgid ""
-msgstr ""
+msgstr "如果您在家庭互联网连接后面运行未注册的服务器，您可能需要启用 [端口转发](#port-forwarding)，如下所述。"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:72
 msgid "### 3. Directory"
-msgstr ""
+msgstr "### 3. 目录"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:74
 msgid "If you want to run a number of Servers, possibly also behind a firewall or on a LAN, you may want to run your Server as a Directory. Examples include online events, music associations, sectional rehearsals or music lessons for schools."
-msgstr ""
+msgstr "如果您想运行多个服务器，也可能在防火墙后面或 LAN 上，您可能希望将您的服务器作为目录运行。示例包括在线活动、音乐协会、部分排练或学校的音乐课。"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:76
 msgid "To run a Directory [read this guide](Directories)"
-msgstr ""
+msgstr "运行目录 [阅读本指南](Directories)"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:79
 msgid "## Installation and Configuration"
-msgstr ""
+msgstr "## 安装和配置"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:81
 msgid "Most people run Jamulus as a \"pure\" Server on **hardware without audio** (e.g. on a 3rd party/cloud host) running Linux. The following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. To run a server on Windows or on the desktop with a graphical user interface, [see this section](#servers-on-the-desktop)."
-msgstr ""
+msgstr "大多数人在运行 Linux 的**没有音频的硬件**（例如，在第 3 方/云主机上）上将 Jamulus 作为“纯”服务器运行。以下步骤假设您熟悉命令行和 Debian/Ubuntu 或使用 systemd 的类似发行版。要在 Windows 或具有图形用户界面的桌面上运行服务器，[请参阅本节](#servers-on-the-desktop)。"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:83
 msgid "If you want to run a Server on a Raspberry Pi (or a different armhf/arm64 debian-based device), you will need to download the `.deb` files for 32 bit `armhf` or 64 bit `arm64`, not the default `amd64` ones you'd use on an Intel/AMD based machine."
-msgstr ""
+msgstr "如果您想在 Raspberry Pi（或其他基于 armhf/arm64 debian 的设备）上运行服务器，您需要下载 32 位 `armhf` 或 64 位 `arm64` 的 `.deb` 文件，而不是您将在基于 Intel/AMD 的机器上使用的默认 `amd64`。"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:86
 msgid "### Installation"
-msgstr ""
+msgstr "### 安装"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:88
 msgid "Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) or, if you use a Raspberry Pi etc. download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or the [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }})"
-msgstr ""
+msgstr "下载 [最新的无头 (amd64) .deb 文件]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) 或者，如果您使用 Raspberry Pi 等，请下载 [最新的 armhf .deb 文件]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) 或 [最新的 arm64 .deb 文件]({{ site.download_root_link }}{{ site.download_file_names.deb-headless- arm64 }})"
```

Comment:
There's an unwanted space in `site.download_file_names.deb-headless- arm64` just before `arm64`.

Reply:
ok